cabby/geo/map_processing/map_test.py

```diff
@@ -23,8 +23,9 @@
 
 class MapTest(unittest.TestCase):
 
-  def setUp(self):
-
+  @classmethod
```

Comment:
Why is @classmethod here?

Reply:
Got, it -- missed the 'Class' bit there. I think you just need setUp() here.

BTW, if you do use setUpClass in future, then use 'cls' instead of 'self'.

Reply:
SetUp runs between before every test and therfore it takes a long time

Reply:
Okay, makes sense! In that case, please update to use 'cls' instead of 'self'.

Reply:
changed to cls